core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -585,17 +612,13 @@ class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers wit
 
         // note: we're copying FROM workspace2 INTO workspace
         assertResult(Seq.empty) {
-          runAndWait(entityQuery.getCopyConflicts(context1, Seq(a1, a2, a3, a4, a5, a6, a)))
+          runAndWait(entityQuery.getCopyConflicts(context1, allEntities.map(_.toReference)))
         }
 
-        assertResult(Seq.empty) {
-          runAndWait(entityQuery.copyEntities(context2, context1, "test", Seq("a1")))
-        }
+        allEntities.map(_.toReference) should contain theSameElementsAs runAndWait(entityQuery.copyEntities(context2, context1, "test", Seq("a1"), false)).entitiesCopied
```

Comment:
This looks like safe usage but please convert these to `assertSameElements` instead